### PR TITLE
Context aware policies can be written with Rego

### PR DESCRIPTION
**Pull request by flavio**
Update the documentation to reflect the possibility to write context aware policies using Rego.

This is part of https://github.com/kubewarden/kubewarden-controller/issues/513

**Note for reviewers:** when looking at the preview site, make sure you switch to the "next" version of the documentation. Otherwise you will not see these changes :wink:

### CODE REVIEW

**Comment by flavio** [comment]
There's some broken css when rendering tabs, see: https://deploy-preview-289--silly-bunny-8cedd0.netlify.app/next/writing-policies/spec/context-aware-policies#rego-policies

I think it would be easier to just drop the tabs and move to dedicated sections:

```md
## Language SDK

### Rust
blah

### Go
blah

## Rego

### Gatekeeper
blah

### OPA

blah
```

What do you think?

**Comment by viccuad** [comment]
I'm ok with dropping the tab. Also, I tend to search and navigate by doing `ctrl+F`, and tabs obscure content.

**Review by viccuad — APPROVED** [review]

**Comment by jhkrug** [comment]
> I'm ok with dropping the tab. Also, I tend to search and navigate by doing `ctrl+F`, and tabs obscure content.

Ditto

File: docs/explanations/context-aware-policies.md [hunk 1]
```diff
@@ -94,3 +101,8 @@ For a core resource, the first will not work, the second will.
 All other Kubernetes resources need the full definition: `<api>/<version>`.
 
 :::
+
+## Further readings
+
+More detailed information about context aware policies can be found inside of [this section](../writing-policies/spec/05-context-aware-policies.mdx)
```
Comment (jhkrug) [inline docs/explanations/context-aware-policies.md:107]:
```suggestion
More detailed information about context aware policies can be found in [this section](../writing-policies/spec/05-context-aware-policies.mdx)
```

File: docs/writing-policies/rego/01-intro-rego.md [hunk 2]
```diff
@@ -7,11 +7,13 @@ title: ""
...
 Rego support has been introduced starting from these releases:
-  
-  * kwctl: v0.2.0
-  * policy-server: v0.2.0
-:::
 
+- kwctl: v0.2.0
+- policy-server: v0.2.0
+
+Rego policies support context aware data stating from the Kubewarden 1.9 release.
```
Comment (jhkrug) [inline docs/writing-policies/rego/01-intro-rego.md:14]:
```suggestion
Rego policies support context aware data starting from the Kubewarden 1.9 release.
```

File: docs/writing-policies/rego/01-intro-rego.md [hunk 3]
```diff
@@ -7,11 +7,13 @@ title: ""
...
+
+:::
 
 The Rego language is a tailor made language designed to embrace
```
Comment (jhkrug) [inline docs/writing-policies/rego/01-intro-rego.md:18]:
```suggestion
The Rego language is a domain specific language designed to embrace
```

File: docs/writing-policies/spec/05-context-aware-policies.mdx [hunk 4]
```diff
@@ -17,20 +17,26 @@ Access to Kubernetes is regulated by RBAC rules applied to the Service Account u
...
-
-:::info
-Access to Kubernetes resources is currently available only to Kubewarden policies written using traditional programming languages. This is not available yet for Rego-based policies.
-:::
+- Namespaces
+- Services
+- Ingresses
 
 :::caution
 The policy server performs caching of the results obtained from the Kubernetes API server to reduce the amount of load of this core piece of Kubernetes.
```
Comment (jhkrug) [inline docs/writing-policies/spec/05-context-aware-policies.mdx:25]:
```suggestion
The policy server performs caching of the results obtained from the Kubernetes API server to reduce the amount of load on this core part of Kubernetes.
```

**Review by jhkrug — CHANGES_REQUESTED** [review]
Some minor suggestions.

**Comment by flavio** [comment]
I've removed the tabgroup. There's a DCO failure right now, I'll fix it before merging the PR

**Review by jhkrug — APPROVED** [review]
lgtm

Also, I'll be going through these again in a while as part of my review tasks.

**Review by fabriziosestito — APPROVED** [review]

**Pull request merged**